src/SFA.DAS.ApplyService.InternalApi/Controllers/RoatpAssessorController.cs

```diff
@@ -149,12 +149,99 @@ private static string GetSequenceTitle(int sequenceId)
                     return null;
             }
         }
+
+        [HttpPost("Assessor/SubmitPageOutcome")]
+        public async Task SubmitAssessorPageOutcome([FromBody] SubmitAssessorPageOutcomeRequest request)
+        {
+            await _mediator.Send(new SubmitAssessorPageOutcomeHandlerRequest(request.ApplicationId,
```

Comment:
What's the value in creating one request type from another? The handler could use SubmitAssessorPageOutcomeRequest?

Reply:
I couldn’t see it either, but then I saw that this is the approach taken in AssignApplication() method.
It uses AssignAssessorApplicationRequest and AssignAssessorRequest. (applicationId could be included in AssignAssessorApplicationRequest and they will become the same)
Plus, …HandlerRequests inherit from IRequest<T>. The other ones don’t. 
And I have decided to follow the same pattern.

Reply:
The two request objects are merged into one.